content/en/docs/contribute/generate-ref-docs/kubernetes-components.md

```diff
@@ -20,14 +20,15 @@ reference documentation for tools and components in the
     * [Python 3.7.x](https://www.python.org/downloads/)
     * [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
     * [Golang](https://golang.org/doc/install) version 1.12 for Kubernetes 1.14+; Go 1.13 [is not supported](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#go)
-    * [Pip](https://pypi.org/project/pip/), which is used to install PyYAML (installed with Python)
+    * [Pip](https://pypi.org/project/pip/), which is used to install PyYAML (not installed by default on Ubuntu)
```

Comment:
@aimeeu, this is a nit: I would rather not see the comment about "... not installed ... on Ubuntu"). It seems clear just to say why Pip is a requirement.

Reply:
@kbhawkey I agree. I removed that yesterday early afternoon. Hmmmmm.... it was after I rebased to remove a merge conflict. I have no idea why the doc change didn't get to pushed to the PR when I pushed.

Reply:
Wow -- there are a lot of commits on the last commit. Did you rebase, something like, git fetch upstream,
git rebase -i upstream/master?

Reply:
:grimacing:

Reply:
I am not sure what happened, but the second commit seems to contain the latest change to that file?
At this pt, it may be worthwhile to get these changes in and correct the few remaining things in a second PR.

Reply:
Yeah, I'm abandoning this PR after I incorporate changes into a new one. I tried to revert the merge commit, which ended in total disaster.

Reply:
Why not merge this PR?

Reply:
I'm not able to squash the commits. However, you are an approver so if you want to merge it, that's fine. Otherwise, I have a new PR ready to go. Let me know what you prefer.

Reply:
Since I have no idea what happened, I suppose you should create another PR.
Based upon what I see in the preview and the source, the changes seem fine.